spec/game_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'game'
+
+describe Game do
+  
+  it 'calculates a gutter game' do
+    game = Game.new
```

Comment:
You don't need to say "game = Game.new" at the beginning of each statement. You can either declare this once at the top by saying "let(:game) { Game.new}", or because you're not initiating with any arguments, you can just use the word "subject" instead of "game" as in "20.times { subject.roll(0) }"